README.md

```diff
@@ -352,7 +357,7 @@ status:
 
 ## Reconciler Implementation
 
-A Reconciler implementation for the `ServiceBinding` type is responsible for binding the Provisioned Service binding `Secret` into an Application.  The `Secret` referred to by `.status.binding.name` on the resource represented by `service` **MUST** be mounted as a volume on the resource represented by `application`.  If the `application` resource is managed by another Reconciler, a `ServiceBinding` Implementations **SHOULD** ensure that the `Secret` volume mount configuration remains after the other Reconciler completes.
+A Reconciler implementation for the `ServiceBinding` type is responsible for binding the Provisioned Service binding `Secret` into an Application.  The `Secret` referred to by `.status.binding.name` on the resource represented by `service` **MUST** be mounted as a volume on the resource represented by `application`.  If the `application` resource is managed by another Reconciler it is **RECOMMENDED** that both Reconcilers comply with the specification extension defined in [Custom Projection](#custom-projection).  
```

Comment:
not sure I agree with that statement.   I actually think it's important to make clear whenever an extension is available to expand on the current core context.   Extensions are optional functionality, but they are still in the spec.

Reply:
You end up with logical circular references if you start doing this.  The core spec *must* be able to stand on its own, without any knowledge of an extension.  If we violate this, it would require implementations that are opting _not_ to implement an extension to have knowledge and work around that extension.

Reply:
I think the key is that this text is _recommending_ the extension for a particular scenario, not _mandating_ it.  So implementers of the core spec can happily ignore that recommendation as if it wasn't there.  But, they could also like the recommendation and look further into the extension for a more compatible implementation.

Reply:
Was it intentional to remove this line?

> `ServiceBinding` Implementations **SHOULD** ensure that the `Secret` volume mount configuration remains after the other Reconciler completes.

The way the new sentence is spliced is a bit off:

> If the `application` resource is managed by another Reconciler it is **RECOMMENDED** that both Reconcilers comply with the specification extension defined in [Custom Projection](#custom-projection).

The first half is talking about how the application resource's desired state is defined, the second half (custom projection) is about how the application resource's desired state is reconciled.

I would still prefer that we avoid links to extensions in the spec's core. If we want to hint to the user that there is more on the subject in an extension we can say something like:

> Specification extensions may define more specific projection behavior.

Reply:
> Was it intentional to remove this line?

Yeah, it hints at an anti-pattern.  If implementations want to do this that's fine, but the spec shouldn't advocate for it. :) 

Thanks for the feedback on the extensions - I will rework that portion in a new commit.

Reply:
In the new commit I just remove that sentence - this way the paragraph becomes more concise and stays within the core behaviour.